pcsx2/R5900OpcodeImpl.cpp

```diff
@@ -670,8 +670,8 @@ static const u64 LDL_MASK[8] =
 	0x0000000000ffffffLL, 0x000000000000ffffLL, 0x00000000000000ffLL, 0x0000000000000000LL
 };
 static const u64 LDR_MASK[8] =
-{	0x0000000000000000LL, 0xff00000000000000LL, 0xffff000000000000LL, 0xffffff0000000000LL,
-	0xffffffff00000000LL, 0xffffffffff000000LL, 0xffffffffffff0000LL, 0xffffffffffffff00LL
+{	0x0000000000000000, 0xff00000000000000, 0xffff000000000000, 0xffffff0000000000,
```

Comment:
I'm not sure it is legual to have a 64 bits constant without the long attribute. I'm nearly sure it won't work on linux.